extensions/amp-instagram/1.0/base-element.js

```diff
@@ -30,8 +30,8 @@ BaseElement['loadable'] = true;
 
 /** @override */
 BaseElement['props'] = {
-  'shortcode': {attr: 'data-shortcode'},
-  'captioned': {attr: 'data-captioned'},
+  'shortcode': {attr: 'data-shortcode', media: true},
```

Comment:
Ditto. I think applying media here is a stretch.

Reply:
Captions too or just shortcode?

Reply:
I thought both. But I actually don't know for "captions" really do.

Reply:
`data-captioned`:
> Include the Instagram caption. amp-instagram will attempt to resize to the correct height including the caption.

Reply:
I see. So somewhat similar issue: SSR and client might disagree and cause the reflow. With the height, it'd also affect other elements on the page.